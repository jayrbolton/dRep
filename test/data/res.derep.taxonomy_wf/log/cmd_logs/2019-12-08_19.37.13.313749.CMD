/usr/local/bin/checkm taxonomy_wf domain Bacteria /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/prodigal/ /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/checkM/checkM_outdir/ -f /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/checkM/checkM_outdir//results.tsv --tab_table -t 6 -g -x faa
