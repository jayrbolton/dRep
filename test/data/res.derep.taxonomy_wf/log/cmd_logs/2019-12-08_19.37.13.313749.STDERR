    Finished processing 0 of 8 (0.00%) bins.    Finished processing 1 of 8 (12.50%) bins.    Finished processing 2 of 8 (25.00%) bins.    Finished processing 3 of 8 (37.50%) bins.    Finished processing 4 of 8 (50.00%) bins.    Finished processing 5 of 8 (62.50%) bins.    Finished processing 6 of 8 (75.00%) bins.    Finished processing 7 of 8 (87.50%) bins.    Finished processing 8 of 8 (100.00%) bins.
    Finished parsing hits for 1 of 8 (12.50%) bins.    Finished parsing hits for 2 of 8 (25.00%) bins.    Finished parsing hits for 3 of 8 (37.50%) bins.    Finished parsing hits for 4 of 8 (50.00%) bins.    Finished parsing hits for 5 of 8 (62.50%) bins.    Finished parsing hits for 6 of 8 (75.00%) bins.    Finished parsing hits for 7 of 8 (87.50%) bins.    Finished parsing hits for 8 of 8 (100.00%) bins.
    Finished processing 0 of 8 (0.00%) bins.    Finished processing 1 of 8 (12.50%) bins.    Finished processing 2 of 8 (25.00%) bins.    Finished processing 3 of 8 (37.50%) bins.    Finished processing 4 of 8 (50.00%) bins.    Finished processing 5 of 8 (62.50%) bins.    Finished processing 6 of 8 (75.00%) bins.    Finished processing 7 of 8 (87.50%) bins.    Finished processing 8 of 8 (100.00%) bins.
    Finished processing 0 of 8 (0.00%) bins.    Finished processing 1 of 8 (12.50%) bins.    Finished processing 2 of 8 (25.00%) bins.    Finished processing 3 of 8 (37.50%) bins.    Finished processing 4 of 8 (50.00%) bins.    Finished processing 5 of 8 (62.50%) bins.    Finished processing 6 of 8 (75.00%) bins.    Finished processing 7 of 8 (87.50%) bins.    Finished processing 8 of 8 (100.00%) bins.
    Finished parsing hits for 1 of 8 (12.50%) bins.    Finished parsing hits for 2 of 8 (25.00%) bins.    Finished parsing hits for 3 of 8 (37.50%) bins.    Finished parsing hits for 4 of 8 (50.00%) bins.    Finished parsing hits for 5 of 8 (62.50%) bins.    Finished parsing hits for 6 of 8 (75.00%) bins.    Finished parsing hits for 7 of 8 (87.50%) bins.    Finished parsing hits for 8 of 8 (100.00%) bins.
